e2etest/pkg/frr/container/container.go

```diff
@@ -107,13 +107,18 @@ func startContainer(cfg Config, testDirName string) error {
 		return errors.Wrapf(err, "Failed to copy FRR config directory. %s", string(res))
 	}
 
+	coreFilePattern := frrMountPath + "/core-%e.%p.%h.%t"
+	res, err = exec.Command("echo", coreFilePattern, "|", "sudo", "tee", "/proc/sys/kernel/core_pattern").CombinedOutput()
```

Comment:
Can we do this in CI? Not sure what will happen if we run it on a host where we are not root (i.e. ocp CI)

Reply:
Also, we would be doing this one time for container, whereas we can do it only once.